### PR TITLE
Add io.posidon.Paper

### PR DESCRIPTION
**Pull request by zaguragit**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by leoxshn** [comment]
I'm not sure if I should include the metadata files, or if the bot will gather them automatically from the source...

**Comment by leoxshn** [comment]
bot, build io.posidon.Paper

File: io.posidon.Paper.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "io.posidon.Paper",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
```
Comment (hfiguiere) [inline io.posidon.Paper.json:4]:
```suggestion
    "runtime-version" : "41",
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: io.posidon.Paper.json (hunk 1 above)
Comment (leoxshn) [reply io.posidon.Paper.json:4]: Can't do it, cuz the app needs libadwaita, I guess I'll just wait for 42

Comment (hfiguiere) [reply io.posidon.Paper.json:4]: you can also build libadwaita like other packages have done.

**Pull request closed without merging**